db/database.go

```diff
@@ -47,10 +47,11 @@ var RunStateString = []string{
 }
 
 const (
-	DefaultRevsLimit = 1000
-	KSyncKeyPrefix   = "_sync:"         // All special/internal documents the gateway creates have this prefix in their keys.
-	kSyncDataKey     = "_sync:syncdata" // Key used to store sync function
-	KSyncXattrName   = "_sync"          // Name of XATTR used to store sync metadata
+	DefaultRevsLimit     = 1000
+	DefaultPurgeInterval = 30               // Default metadata purge interval, in days.  Used if server's purge interval is unavailable
```

Comment:
This is a bit confusing to have a default of 30 days here and a `DefaultMetadataPurgeInterval = 72`.  Can you add some comments to explain the different defaults?